packages/maskbook/src/plugins/Collectible/UI/ListingByHighestBidCard.tsx

```diff
@@ -146,13 +147,11 @@ export function ListingByHighestBidCard(props: ListingByHighestBidCardProps) {
                 <DateTimePanel
                     label="Expiration Date"
```

Comment:
i18n

Reply:
Would patch it in another PR.

Reply:
Already got patched in https://github.com/DimensionDev/Maskbook/pull/3228